event_registration_mass_confirm/wizard/confirm_attendees.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import models, api, _
+
+
+class EventRegistrationMassConfirm(models.TransientModel):
+
+    _name = "event.registration.massconfirm"
+
+    @api.multi
+    def confirm_registrations(self):
+        """Validate selected registrations."""
+        context = dict(self.env.context or {})
```

Comment:
`context = self.env.context.copy()`

Reply:
hmm wait, why do you need to copy it? you are just getting values from it. Also, you should pass it in the returned action.